web-ts/tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "lib": ["dom", "dom.iterable", "ES5", "ES6"],
     "noEmit": false,
     "outDir": "dist",
+    "sourceMap": true,
```

Comment:
This allwos to debug the webapp in typescript directly. Should I write some documentation for this?

Reply:
I think that would be nice, yes!

Reply:
Maybe keep in mind that there is currently an open documentation PR to avoid merge conflicts: #340